### PR TITLE
YLP-2065 Migrate CBGDateTraceLabel from viz to dumb

### PR DESCRIPTION
**Pull request by TimCanu**
## Checklist
- [x] PR name contains the Jira ticket number
- [x] Changelog updated
- [x] Code tested (unit tests and integration tests)
- [x] No console errors/warnings in integration tests
- [x] No translation files updated (except for English files)
- [x] Code formatted on every file
- [x] Copyright added on every file
- [x] End-to-end tests successful on CI

### CODE REVIEW

File: packages/dumb/src/models/focuse-date-trace.model.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+export interface FocuseDateTrace {
```
Comment (fatb38) [inline packages/dumb/src/models/focuse-date-trace.model.ts:28]: Missing a d
```suggestion
export interface FocusedDateTrace {
```

File: packages/dumb/src/components/trends/cbg/cbg-date-trace/cbg-date-trace-label.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+.container {
+  composes: absolute smallHorizontalPadding from '../../../../styles/layout.css';
+  top: 0px;
```
Comment (fatb38) [inline packages/dumb/src/components/trends/cbg/cbg-date-trace/cbg-date-trace-label.css:30]:
```suggestion
  top: 0;
```

File: packages/dumb/src/components/trends/cbg/cbg-date-trace/cbg-date-trace-label.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+interface CbgDateTraceLabelProps {
+  focusedDateTrace: FocusedDateTrace
+}
+
+export const CbgDateTraceLabel: FunctionComponent<CbgDateTraceLabelProps> = (props) => {
+  const { focusedDateTrace } = props
+
+  const { t } = useTranslation()
+
+  const formattedDate = moment.utc(focusedDateTrace.data.localDate).format(t('dddd, MMMM D'))
```
Comment (Nas84) [inline packages/dumb/src/components/trends/cbg/cbg-date-trace/cbg-date-trace-label.tsx:45]: Should we reuse formatLocalizedFromUTC  (from datetime.utils) ?

**Pull request merged**